src/UI/LoginButton/LoginButton.tsx

```diff
@@ -10,13 +10,15 @@ type LoginButtonProps = {
   customClassName?: string;
   btnClassName?: string;
   children?: React.ReactNode;
+  disabled?: boolean;
 };
 
 export const LoginButton = ({
   onLogin,
   text = 'Default Login Button',
   customClassName = '',
   btnClassName = 'default-login-button',
+  disabled,
```

Comment:
suggestion: lets stick with the "is/should" prefix for booleans naming convenstions